### ISSUE TITLE
Refactoring: remove Lua path-related duplicate code in **/test.sh and **/init.lua

### ISSUE THREAD

**Issue by bofm** [moonlibs/tarantoolapp#5]
There is the same code piece in Bash repeated in test.sh in basic/ and vshard/ templates. Also there seems to be the same code in Lua in **/init.lua. Those pieces of code are related to Lua path and cpath definition. It would be great if that code was moved to a single place to avoid copy-paste.

**Comment by igorcoding** [comment]
I agree on that, currently I'm thinking of a better way to add paths to make it relatively simple and obvious to understand where everything comes from.
